pkg/k8s/crd/Makefile

```diff
@@ -82,19 +82,19 @@ run: manifests generate fmt vet ## Run a controller from your host.
 	go run ./main.go
 
 docker-build: test ## Build docker image with the manager.
-	docker build -t ${IMG} .
+	cd ../../../ && docker build -f pkg/k8s/crd/Dockerfile -t ${IMG} .
```

Comment:
do we need this if we execute docker-build from `shifu`'s root dir?

Reply:
No, I only added those lines because this command is run in pkg/k8s/crd